src/language/en-US.json

```
@@ -1087,5 +1089,11 @@
     "import_payload": "Import payload",
     "import_payload_text": "You can also enter transaction payloads generated by symbol-cli",
     "ledger_available_on_standalone_app": "To use Ledger please download the desktop native app.",
-    "delegated_harvesting_request_failed": "Kindly note that your delegated harvesting request has failed, please stop harvesting and try a different node."
+    "select_custom_node": "Select custom node",
+    "delegated_harvesting_request_failed": "Kindly note that your delegated harvesting request has failed, please stop harvesting and try a different node.",
```

I don't think it's usual for a program to be so polite. I'd use something like:

```suggestion
    "delegated_harvesting_request_failed": "Your delegated harvesting request has failed, please stop harvesting and try a different node.",
```